content/blog/2024-06-20-why-rse.md

```diff
@@ -0,0 +1,100 @@
+
+Independently of their area of expertise, their common goal is better maintainability, sustainability of research software and better reproducibility of research outputs.
+
+**What an RSE is not**
+
+Even though they are in close collaboration with IT services, RSEs are not IT support people, as in IT manager or IT system specialists. They have a dual background (research/coding), they take part in research projects and speak the "same language" as researchers. 
+
+They also are not enforcers, they don't enforce technologies or methods but build on the tools already in use in research groups.  
+
+#### Why would an organization need RSEs? - Current state of Research software 
+
+Does any of following scenario sound familiar? 
+
+* A PhD/post doc has developed some really nice piece of code, but they left and no one knows anymore how to run it or develop it further. 
+* Someone wrote some code long time ago that is used daily by a research group, but no one knows the technology in the research group anymore and it cannot be modified, updated or compiled anymore. 
+* When a newcomer joins a team and starts familiarizing with analysis/modelling/etc tools, the only way is to "ask others who use it". No document or tutorials are available. 
+* Some essential piece of software runs on a very old machine or on very old version of language and it's very difficult to replace or update, if even possible. This can even lead as far as stopping research activities in the team for some time 
+* After an article is submitted, one reviewer ask for additional analysis, or slightly different plots, but upon running the code again, one cannot reproduce the plots/results anymore. 
+
+#### How RSE can help ? 
+
+**Short answer**
+
+Improve FAIRness of research software and by that save time and resources. 
+
+**Long answer**
+
+RSE can leverage all the above issues and more by accompanying researchers in their software development journey. RSEs' technical knowledge and creativity on solving problems allow seeing where researchers would benefit the most. Here are a few examples/use cases: 
+
+* Researchers have great ideas, but might not know how, or have time to develop the corresponding tools 
+* Help to get rid of repetitive or time consuming steps (that users don't want to do themselves) by introducing automatization 
+* Write documentation so the information doesn't get lost when people leave
+* Make the regular technology updates to prevent that it become obsolete and break
+* Supporting when switching from laptop to compute clusters 
+* Many groups need the same thing, RSEs know how to create flexible templates to simplify e.g. spinning up a website to present research outputs
+* … what other good examples? 
```

Comment:
I would rather bring FAIR software in here (+ publication of software) as one point here

Reply:
Suggestion added above